src/ipc/subscribe-to-devices.ts

```diff
@@ -0,0 +1,29 @@
+import { IpcMain, IpcMainInvokeEvent } from 'electron'
+import usbDetection from 'usb-detection'
+
+export const channel = 'subscribe-to-devices'
+export const deviceUpdateChannel = 'device-update'
+
+export enum UpdateType {
+  Add,
+  Remove,
+}
+
+/**
+ * Subscribe to add/remove USB device events.
+ */
+export default function register(ipcMain: IpcMain): void {
+  ipcMain.handle(channel, (event: IpcMainInvokeEvent) => {
+    const webContents = event.sender
+
+    usbDetection.startMonitoring()
```

Comment:
pedantic, but does it matter if you `startMonitoring` before you add event handlers? I know, JS event loop etc, just wondering if it is easier to read if you register event handlers first.

Reply:
FWIW that's what the [README example does](https://github.com/MadLittleMods/node-usb-detection).

Reply:
Fair enough!

Reply:
I switched it anyway. Works either way because it's actually an `EventEmitter`.